homeassistant/components/wallbox/const.py

```diff
@@ -21,5 +21,6 @@
 CONF_LOCKED_UNLOCKED_KEY = "locked"
 CONF_NAME_KEY = "name"
 CONF_STATE_OF_CHARGE_KEY = "state_of_charge"
+CONF_STATUS_ID_KEY = "status_id"
```

Comment:
Many of these constants that are prefixed with `CONF_` don't seem to represent configuration. We only use `CONF_*` constants for configuration items.